### ISSUE TITLE
Accessibility: Landing page missing language attribute [prea-003]

### ISSUE THREAD

**Issue by aaron-collier** [sul-dlss/pre-assembly#1258]
## Suggestion

Add the lang element (i.e. `<html lang="en">`) to the HTML element.

https://docs.google.com/spreadsheets/d/105GozrKuWrnXaEk5FoP4UbzHSCTPgeEqPaPiHi1c950/edit#gid=0
